README.md

```diff
@@ -10,9 +10,10 @@ ESlint Rules for the Standard Linter
 ```js
 {
   rules: {
-    'standard/object-curly-even-spacing': [2, "either"]
+    'standard/object-curly-even-spacing': [, "either"]
```

Comment:
oh yeah, we should progbably use the fancy new `"error"` syntax here, but yes let me fix!